$MirOS$

	Help sucky operating systems to be less sucky.

	In general, if the command
	$ env -i LC_CTYPE=en_US.UTF-8 /usr/bin/locale >&-
	fails - you can check this for example with
	$ env -i LC_CTYPE=nonexistant /usr/bin/locale >&-
	then you need to take some action, e.g.
	# localedef -c -i en_US -f UTF-8 en_US.UTF-8
	(or with sudo), if that works.

--- subversion/libsvn_repos/repos.c.orig	Wed Aug 23 13:37:02 2006
+++ subversion/libsvn_repos/repos.c	Tue Jan  9 22:29:40 2007
@@ -1038,10 +1038,22 @@ create_hooks(svn_repos_t *repos, apr_poo
       HOOKS_ENVIRONMENT_TEXT
       "# "
       APR_EOL_STR
+      "# Its standard output and standard error are not displayed"
+      APR_EOL_STR
+      "# to the client either."
+      APR_EOL_STR
+      "# "
+      APR_EOL_STR
       "# Here is an example hook script, for a Unix /bin/sh interpreter."
       APR_EOL_STR
       PREWRITTEN_HOOKS_TEXT
       APR_EOL_STR
+      APR_EOL_STR
+      "# If this breaks, your operating system is fucked up anyway."
+      APR_EOL_STR
+      "# Consider the documentation on how to create this locale, then."
+      APR_EOL_STR
+      "export LC_CTYPE=en_US.UTF-8"
       APR_EOL_STR
       "REPOS=\"$1\""
       APR_EOL_STR
